tests/features/general.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Miscellaneous general settings unit tests
+
+  @ubi8/openjdk-8
+  @ubi8/openjdk-8-runtime
+  @ubi8/openjdk-11
+  @ubi8/openjdk-11-runtime
+  @ubi8/openjdk-17
+  @ubi8/openjdk-17-runtime
+  Scenario: Check the attributes of /home/jboss using stat
+    When container is started with args
+    | arg     | value            |
+    | command | stat /home/jboss |
+    Then available container log should contain File:
```

Comment:
substitute File: with one of the lines of output we would expect to see from `stat`, which would prove the permissions are correct, e.g. `Access: (0771/drwxrwx--x)  Uid: (  185/   jboss)   Gid: (    0/    root)`